### PR TITLE
Bump openedx-completion-aggregator

### PR DESCRIPTION
**Pull request by viadanna**
Description
------------
This PR bumps `openedx-completion-aggregator` to fix a serialisation error.

JIRA tickets
------------
MCKIN-14043
MCKIN-14046
MCKIN-14177
MCKIN-14178

Dependencies
----------------
https://github.com/open-craft/openedx-completion-aggregator/pull/68

Testing instructions
--------------------

1. Deploy this to integration or another environment that makes use of Celery tasks
1. Enroll a user in a course
1. Uneroll a user from a course
1. Create a group of students in Group Work
1. Assign students to a Program
1. Register for course access
1. Ensure no failures or `Internal Server Error` messages were generated

Author notes and concerns
----------------------------

1. **Merge deadline**: ASAP
1. There's no way to test this in a devstack yet

Reviewers
-----------

- [x] @Agrendalath 
- [ ] edX reviewer[s] TBD

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @viadanna! I've created [OSPR-4382](https://openedx.atlassian.net/browse/OSPR-4382) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

**Comment by natabene** [comment]
@viadanna Please let me know once it is ready for our review. @ihtram FYI this will be coming your way.

**Comment by Agrendalath** [comment]
@viadanna, I dont' think we should convert `course_key` variable in the signal, as this is a breaking change to all plugins subscribed to this. Instead we should convert it with `six.text_type` just before sending it to the celery task, like we're doing [here](https://github.com/open-craft/openedx-completion-aggregator/blob/master/completion_aggregator/batch.py#L101).

File: requirements/edx/custom.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@ git+https://github.com/edx-solutions/xblock-group-project.git@0.1.2#egg=xblock-g
 -e git+https://github.com/open-craft/xblock-virtualreality.git@v0.1.1#egg=xblock-virtualreality==0.1.1
 
 git+https://github.com/mckinseyacademy/openedx-user-manager-api@v1.2.0#egg=openedx-user-manager-api==1.2.0
-openedx-completion-aggregator==2.0.0
+git+https://github.com/open-craft/openedx-completion-aggregator.git@paulo/fix-cohort-handler#egg=openedx-completion-aggregator==2.0.1
```
Comment (ihtram) [inline requirements/edx/custom.txt:26]: @viadanna Is there a reason to bump version from a specific branch?

Comment (Agrendalath) [reply requirements/edx/custom.txt]: @ihtram, the branch was specified temporarily, to ensure that the change won't break the CI tests. I pushed the commit with the actual version bump.
cc: @viadanna

**Pull request merged**